src/jit/compiler.h

```diff
@@ -6931,6 +6929,9 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     // Should we support SIMD intrinsics?
     bool featureSIMD;
 
+    // Have we identified any SIMD types?
+    bool foundSIMDType;
```

Comment:
You might also want to signal that this is stored as part of the compiler instance.
for both compFeatureSIMD and compFoundSIMDType

When I saw the assignement 

foundSIMDType = true;

later on it didn't jump out to me that this was changing the global state of the compiler instance.

I think that using accessor functions would be best:

    bool usesSIMDType()  { return _usesSIMDType; }
    void setUsesSIMDType() { _usesSIMDType = true; }
    void clearUsesSIMDType() { _usesSIMDType = false; }